hedyorg/hedy
# 📜 Adds workbooks in en and nl

Adds workbooks to pages

**How to test**

Follow these steps to verify this PR works as intended:

* Go to http://127.0.0.1:8080/for-teachers/workbooks/1 and witness the nice worksheets!!

<img width="1312" alt="image" src="https://github.com/user-attachments/assets/594b0814-8c58-4067-b00e-6a5faf9ac560">
